### PR TITLE
Sites yaml for gastby showcase

### PR DESCRIPTION
**Pull request by jpvajda**
## Description
- includes the file for sites.yaml
- includes a new screenshot directory
- includes a screenshot 

## Related Issue(s) / Ticket(s)
If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.

#618

### CODE REVIEW

File: sites.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+- title: New Relic Developer Site
+
+  # this is the URL that is linked from the showcase
+  main_url: https://developer.newrelic.com/
+
+  # this URL is used to generate a screenshot
+  url: https://github.com/newrelic/developer-website/screenshots
```
Comment (jpvajda) [inline sites.yml:7]: I was curious if this needs to reference the actual file in the path, the example they gave didn't require the file name.

**Comment by jerelmiller** [comment]
Are we sure that this `sites.yml` file should live here and not the one in the `gatsby` repo itself? 

From what I understood, I think its this file that needs to be edited: https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml

Then again, I could be wrong since I haven't actually done it yet 😄

**Comment by jpvajda** [comment]
@jerelmiller Oh wait, I think you are right! It was weird, their showcase documentation just disappeared one day, as I think they did some domain combining of .org and .com. So yeah, I'll close this. I'm actually glad it works this way, as I thought if we hosted the file, was gastby pulling that in from some automated process or a plugin.

**Comment by jpvajda** [comment]
@jerelmiller  Can we merge in this file and directory + screenshot? I'll need that for the image in the showcase. As for the sites.yml file, I was thinking we can keep it as a reference file with site. so there is a copy of it.

**Comment by jerelmiller** [comment]
Ya I can do that

**Pull request merged**